styles/index.css

```diff
@@ -0,0 +1,173 @@
+body {
+    background-image: url('/assets/weather-background.jpg');
+    opacity: 70;
+}
+
+#main {
+    display: grid;
+    grid-template-columns: repeat(10, 10%);
+    grid-template-rows: repeat(10, 10%);
+    row-gap: 3vh;
+    column-gap: 3vw;
+}
+
+#gardenBox {
+    grid-column: 2/span 6;
+    display: flex;
+    flex-flow: column nowrap;
+    background-color: azure;
+    border-radius: 2vw;
+    justify-content: center;
+    align-items: center;
+}
+
+#weatherData {
```

Comment:
your weather data goes out of its container when it's rendered. Here is a resource I found that might be helpful when using grid and flex https://css-tricks.com/preventing-a-grid-blowout/